optimum/bettertransformer/models/encoder_models.py

```diff
@@ -82,6 +82,23 @@ def __init__(self, albert_layer, config):
         # Last step: set the last layer to `False` -> this will be set to `True` when converting the model
         self.is_last_layer = False
 
+        self.original_layers_mapping = {
+            "in_proj_weight": ["attention.query.weight", "attention.key.weight", "attention.value.weight"],
+            "in_proj_bias": ["attention.query.bias", "attention.key.bias", "attention.value.bias"],
+            "out_proj_weight": "attention.dense.weight",
+            "out_proj_bias": "attention.dense.bias",
+            "linear1_weight": "ffn.weight",
+            "linear1_bias": "ffn.bias",
+            "linear2_weight": "ffn_output.weight",
+            "linear2_bias": "ffn_output.bias",
+            "norm1_eps": "attention.LayerNorm.eps",
+            "norm1_weight": "attention.LayerNorm.weight",
+            "norm1_bias": "attention.LayerNorm.bias",
+            "norm2_eps": "full_layer_layer_norm.eps",
+            "norm2_weight": "full_layer_layer_norm.weight",
+            "norm2_bias": "full_layer_layer_norm.bias",
+        }
```

Comment:
This sounds more like a class attribute to me, I'd put this not in the init, e.g. with all caps after `class AlbertLayerBetterTransformer`. But anyway this may change in the future so I guess it's fine to leave as is.